application/src/main/resources/db/V13__Add_Reminder_Failure_Counter.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE pending_reminders ADD reminder_failure_counter INTEGER DEFAULT 0;
```

Comment:
* its clear from the context that its a reminder-thing, so id rename it to just `failure_counter`. or, to make it clearer what it actually is, `failed_attempts`
* its missing a `NOT NULL` at the end

Reply:
done